README.md

```diff
@@ -847,6 +847,9 @@ OPTIONS:
     UF2 Family options
         <family_id>
             family ID for UF2
+    Platform options
+        <platform>
+            optional platform for memory verification (eg rp2040, rp2350)
```

Comment:
I dunno if this is a stupid question, but is it possible to infer the `platform` from the `family_id` ?

Reply:
Not always (eg for `absolute` or `data` IDs) - this option allows you to specify it explicitly, if not specified then it does the usual search for a block loop/checksum (see `get_access_model`)